buf/registry/module/v1beta1/upload_service.proto

```diff
@@ -98,3 +98,19 @@ message UploadResponse {
   // If nothing changed for a given reference, the existing Commit will be returned.
   repeated Commit commits = 1 [(buf.validate.field).repeated.min_items = 1];
 }
+
+// UploadErrorDetails is the error details for the Upload RPC.
+message UploadErrorDetails {
+  message DepRef {
+    // The commit_id of the dependency.
+    string commit_id = 1 [
+      (buf.validate.field).required = true,
+      (buf.validate.field).string.tuuid = true
+    ];
+    // The registry hostname of the dependency.
+    string registry = 2 [(buf.validate.field).required = true];
+  }
+  // This contains the dep_refs that the server is unable to find. This is populated
+  // when a non-found error is returned by the server.
+  repeated string not_found_dep_refs = 1;
```

Comment:
Is this supposed to be `repeated DepRef`?

Reply:
Yes, fixed in [da730f2](https://github.com/bufbuild/registry-proto/pull/129/commits/da730f285913e9b777a7653d11e29678711b12b7)